SellerMarket/locustfile_new.py

```
@@ -446,8 +446,48 @@ def on_test_stop(environment, **kwargs):
+                                    volume = order.get('volume', 0)
+                                    executed = order.get('executed_volume', 0)
+                                    
+                                    order_summaries.append(
+                                        f"• {symbol}: {tracking_number} ({executed}/{volume}) - {state_desc}"
+                                    )
+                                
+                                account_details.append(
+                                    f"👤 *{username}@{broker_code}:*\n" + "\n".join(order_summaries)
+                                )
+                except Exception as e:
+                    logger.exception(f"Error getting details for {username}@{broker_code}: {e}")
```

Ruff (0.14.3)

Redundant exception object included in `logging.exception` call

(TRY401)